site/src/pages/docs/docs-layout.tsx

```diff
@@ -3,6 +3,16 @@ import { globalStyle, style } from '@macaron-css/core';
 import { styled } from '@macaron-css/react';
 import { Link } from '../../../renderer/Link';
 
+globalStyle('blockquote', {
+  padding: '0.8rem',
+  borderLeft: '2px solid #ff4089',
+  background: '#ffffff10',
+});
+
+globalStyle('blockquote p', {
+  fontSize: '0.8rem !important',
```

Comment:
Increase the font size to be the same as rest

Reply:
It doesn't look that good that way
This is normal font size:
![Screenshot 2022-12-06 at 14 12 51](https://user-images.githubusercontent.com/41034356/205862573-12df6016-c1cd-417b-b215-16833af8a8e4.png)
This is `0.8rem`:
![Screenshot 2022-12-06 at 14 13 19](https://user-images.githubusercontent.com/41034356/205862670-6de9309f-d562-4dc0-917b-8e3d3a74f735.png)

Reply:
hm, `0.8rem` looks too small. maybe `1rem`?

Reply:
![Screenshot 2022-12-06 at 14 39 00](https://user-images.githubusercontent.com/41034356/205868530-b654cb44-cb1d-4843-ab1e-14bb6ba7c743.png)

Reply:
yeah this looks good